### PR TITLE
add macos beacon node

### PR DESCRIPTION
**Pull request by arthurk**
add macos beacon nodes

### CODE REVIEW

File: prater.tf [hunk 1]
```diff
@@ -184,3 +184,17 @@ module "nimbus_nodes_prater_hetzner" {
 
   ips = ["65.21.73.183"]
 }
+
+module "nimbus_nodes_prater_macos" {
+  source = "./modules/hetzner"
```
Comment (jakubgs) [inline prater.tf:189]: This module should be renamed to something like `dummy-host` or something, since it's being used for non-hetzner host.

Comment (arthurk) [reply prater.tf]: there is a `mac-stadium` module, i forgot to change it in the previous commit

File: ansible/prater.yml [hunk 2]
```diff
@@ -91,3 +91,27 @@
...
+      tags: [ beacon-node, infra-role-beacon-node-macos ]
+      vars:
+        beacon_node_repo_branch: '{{ node.branch }}'
+        beacon_node_build_frequency: '{{ node.build_frequency }}'
+        beacon_node_discovery_port: '{{ 9000 + port_offset }}'
+        beacon_node_listening_port: '{{ 9000 + port_offset }}'
+        beacon_node_metrics_port:   '{{ 9200 + port_offset }}'
+        beacon_node_rpc_port:       '{{ 9900 + port_offset }}'
+        beacon_node_dist_validators_start: '{{ node.start }}'
+        beacon_node_dist_validators_end: '{{ node.end }}'
```
Comment (jakubgs) [inline ansible/prater.yml:110]: You can explicitly disable this for now with `beacon_node_dist_validators_enabled: false`.

**Review by jakubgs — APPROVED** [review]
Looks good. The bootstrap role does not handle Consul setup yet correct?

**Comment by arthurk** [comment]
yes correct

**Review by jakubgs — CHANGES_REQUESTED** [review]
Inventory in the repo was not updated.

**Review by jakubgs — APPROVED** [review]
Please squash this and resolve the conflict and I'll merge this.

**Pull request merged**